.golangci.yml

```diff
@@ -6,3 +6,4 @@ linters:
   enable:
     - goimports
     - gofmt
+    - gosec
```

Comment:
There was some discussion around this in https://github.com/code-ready/crc/pull/568 and https://github.com/code-ready/crc/issues/563 I don't know if this is still relevant

Reply:
Oh I didn't know. 

The main argument was `gosec linter which is part of golangci-lint coming from github.com/golangci/gosec instead github.com/securego/gosec https://github.com/golangci/golangci-lint/blob/master/pkg/golinters/gosec.go#L11 , which is last updated on 2nd Feb https://github.com/golangci/gosec/tree/02.2019 ( https://github.com/golangci/golangci-lint/blob/master/go.mod#L23) which means this is out dated.` but today it's fixed. golangci-lint use the upstream gosec and the latest version.

Reply:
So new version of golang-ci does have fix for gosec and now it's taking upstream codebase instead own downstream bits.